app/api/public-cloud/aws-roles/routes.ts

```diff
@@ -144,7 +172,7 @@ const findObjectByValueSubstring = (array: Group[], key: keyof Group, value: any
   return array.filter((obj) => obj[key].includes(value));
 };
 
-export async function getProductAWSRoles(licencePlate: string = 'eu9cfk'): Promise<Group[] | undefined> {
+export async function getProductAWSRoles(licencePlate: string): Promise<Group[] | undefined> {
   const keyClockGroups = await getGroups;
```

Comment:
The usage seems little bit interesting... maybe we can convert `getGroups` as an actual function?

Reply:
could you, please, provide more explanations with that, thank you

Reply:
yes, can you convert `getGroups` as a `Function`, it is a `Promise` type now.